crates/turborepo-lib/src/config/turbo.rs

```diff
@@ -31,7 +31,7 @@ pub struct TurboJson {
     pub(crate) global_deps: Vec<String>,
     pub(crate) global_dot_env: Vec<RelativeUnixPathBuf>,
     pub(crate) global_env: Vec<String>,
-    pub(crate) global_pass_through_env: Vec<String>,
+    pub(crate) global_pass_through_env: Option<Vec<String>>,
```

Comment:
This is a quirk of Go. Specifically `[]string` is equivalent to `Option<Vec<String>>` in Rust because `[]string` can be `nil` or an empty slice. In the `nil` case, the field is not serialized at all, while in the empty case it is serialized as an empty array.